recipe/build.sh

```diff
@@ -31,6 +31,13 @@ fi
 
 echo ESMF_F90COMPILEOPTS=${ESMF_F90COMPILEOPTS}
 
+if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" == "1"  && "${target_platform}" == "linux-aarch64" ]]; then
+  # having trouble with -m64 on aarch64 when cross-compling and this is the "fix"
+  # that happens when not cross-compiling
+  export ESMF_ABI=32
+fi
```

Comment:
I've got to say I'm not very happy with this fix (let's see if it even works) but this seems to be why we weren't seeing this problem when we weren't cross-compiling.  ESMF seems to think aarch64 is 32 bit. See a recent successful Travis build:
https://app.travis-ci.com/github/conda-forge/esmf-feedstock/jobs/590175359